src/app/analytic-events-handler.ts

```diff
@@ -0,0 +1,71 @@
+export interface IAnalyticData {
```

Comment:
refactor the filename to `analytics-handler`, no need to have `events` in the middle.